arvo/hoon.hoon

```diff
@@ -10079,7 +10061,16 @@
     ++  wisp                                            ::  core tail
       %-  ulva
       %+  cook
-        |=(a/(list {p/term q/foot}) (~(gas by *(map term foot)) a))
+        |=  a/(list {p/term q/foot})
+        =|  take/(set term)
+        =+  cur=a
+        |-
+          ?~  cur
+            (~(gas by *(map term foot) a))
```

Comment:
gas~by syntax 2017

On Friday, 29 July 2016, chc4 <notifications@github.com> wrote:

> In arvo/hoon.hoon
> <https://github.com/urbit/arvo/pull/199#discussion_r72880781>:
>
> > @@ -10079,7 +10061,16 @@
> >      ++  wisp                                            ::  core tail
> >        %-  ulva
> >        %+  cook
> > -        |=(a/(list {p/term q/foot}) (~(gas by *(map term foot)) a))
> > +        |=  a/(list {p/term q/foot})
> > +        =|  take/(set term)
> > +        =+  cur=a
> > +        |-
> > +          ?~  cur
> > +            (~(gas by *(map term foot) a))
>
> Looks like that's correct! No clue why the nest-fail was in such a weird
> place due to it, though.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/urbit/arvo/pull/199/files/9a1830065d28917ffe9ca5d0344798c502409ef0#r72880781>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABxXhjlVOs3OuQdSJtuk0AQZH0BcCPDrks5qar4cgaJpZM4JKm_w>
> .
>